Makefile

```diff
@@ -17,6 +17,7 @@ DOCKER_PUSH?=false
 IMAGE_NAMESPACE?=quay.io/argoproj
 VERSION?=latest
 BASE_VERSION:=latest
+DO_BUILD?=true
```

Comment:
Yes.
In the github release workflow, we do the build separately to avoid the timeout of the quay/docker login, and also have the binaries handy to attach to the release page. In a manual build, we expect the image build order to build the binaries by default.

Reply:
`DO_BUILD` is only used at:

```
image-multi: set-qemu $(if $(DO_BUILD), dist/$(BINARY_NAME)-linux-arm64 dist/$(BINARY_NAME)-linux-amd64)
```

Even `DO_BUILD` is true, if the binaries are already in place, it will not rebuild them. So to my understanding, it's useful at all. Correct me if I'm wrong.

Reply:
I did the test outside of github actions, you are right. So I've clean this.